.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,25 @@
+v### Description
+
+[Description of the bug or feature]
+
+### Steps to Reproduce
+
+1. [First Step]
+2. [Second Step]
+3. [and so on...]
+
+### Expected behavior:
+
+[What you expected to happen]
+
+### Actual behavior:
+
+[What actually happened]
+
+### Version of your dependencies, npm and node
+
+[You can get this information by executing the following command: `npm list --depth=0 && npm -v && node -v`. Just copy and paste it, then hit RETURN in your CLI]
```

Comment:
I'm not sure if it does work for Windows OS. Can you confirm, please?

Reply:
That work, I tried.